beads_web/src/test/java/com/beads/web/dao/TestOrderDaoImpl.java

```diff
@@ -77,6 +88,7 @@ public void testSearchCriteriaWithOrderStatusAndPhoneNumber() {
 
   private void assertOrderBySearchCriteria(Order expectedOrder) {
     List<Order> orders = orderDao.getOrdersBySearchCriteria(searchCriteria);
+    Assert.assertTrue("List orders size must have only one element", orders.size() == 1);
```

Comment:
better to assertHere this as `assertEquals("list size must be as expected", 1, orders.size())`